composer.json

```diff
@@ -237,10 +237,7 @@
         "ext-pcre": "*",
         "ext-json": "*",
         "ext-openssl": "*",
-        "ext-gmp": "*",
```

Comment:
I think BigInteger.php requires either GMP or bcmath...

Reply:
yep we have `ext-bc` already, that's why i removed gmp